tests/rt.conf

```diff
@@ -403,6 +403,9 @@ RUN | cpld_control_nowave_noaero_p8                     | + hera cheyenne hercul
 COMPILE | s2swa_debug | gnu | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | + hera cheyenne hercules | fv3 |
 RUN | cpld_debug_p8                                     | + hera cheyenne hercules            | baseline |
 
+COMPILE | s2sa | gnu | -DAPP=S2SA -DCCPP_SUITES=FV3_GFS_v17_coupled_p8 | + hera cheyenne hercules | fv3 |
```

Comment:
Tests are by default added as intel tests. We test only a limited subset of tests with gnu, so this compile and run should be moved up into the intel section of rt.conf